data/ecosystems/t/ton.toml

```diff
@@ -627,6 +763,9 @@ url = "https://github.com/Blockchair/ton-indexer"
 [[repo]]
 url = "https://github.com/boltoshi-labs/bolt"
 
+[[repo]]
+url = "https://github.com/boltoshi-labs/huebel.art"
```

Comment:
how's this using ton?

Reply:
It's a landing page for a TON-based token https://huebel.art/